src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -447,6 +447,9 @@
   <data name="MultiLinePasteDialog.PrimaryButtonText" xml:space="preserve">
     <value>Paste anyway</value>
   </data>
+  <data name="MultiLinePasteDialog.SecondaryButtonText" xml:space="preserve">
+    <value>Strip line breaks</value>
+  </data>

```

Comment:
Also, could you add a comment tag specifying that this is a button label that strips the line breaks and pastes the presented text? Recently got feedback from the localization team that we should add more comments in and I'm now always on edge hahaha 😅

Reply:
Sure thing. I'll add one for the primary button as well if that helps; let me know if the wording is unclear.